test/src/cli/testParallelExecution.js

```diff
@@ -335,6 +337,23 @@ describe('test Parallel Execution', function() {
 
     runner.setup();
     const worker = runner.concurrency.createChildProcess('test-worker');
+    console.log(worker.settings.selenium);
+
+  });
+
+  it('test random port assignment for parallel execution', function(){
+    const client = Nightwatch.createClient({
+      webdriver: {
+        port: 9999
+      },
+
+      'test_workers': {
+        'enabled': true,
+        'workers': 4
+      }
+    });
+
+    assert.notEqual(client.httpOpts.port, 9999);
   });
```

Comment:
I don't think this tests the changes as for the changes start_process should be `true`.

Reply:
basically it does not matter, I am checking whether the option gets set correctly or not. The option is not dependant on start_process right now. But adding that in for future regression.